wherehows-dao/src/main/java/wherehows/models/DatasetFieldEntity.java

```diff
@@ -18,7 +18,7 @@
   private String identifierType;
   private String identifierField;
   private String logicalType;
-  private Boolean isSubject;
+  private Boolean classification;
```

Comment:
My mistake, should be a String to capture the security classification (confidential, limited_distribution). Fixed, thanks!